build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     ext {
-        springBootVersion = '2.1.0.RELEASE'
+        springBootVersion = '2.0.1.RELEASE'
     }
```

Comment:
Why we need 2.01.RELEASE instead of 2.1.0.RELEASE?

Reply:
Changing to that version solved all the problems I had in the build.